main.py

```diff
@@ -970,32 +1007,32 @@ async def me() -> str:
 
 @mcp_tool(
     api_groups=["release-management"],
-    description="Add a new Release Management connected app to Bitrise."
+    description="Add a new Release Management connected app to Bitrise.",
```

Comment:
Note: we should probably add a common formatter not to keep producing such diffs when different people with different IDEs make changes.